src/main/scala/tilelink/Parameters.scala

```diff
@@ -1390,6 +1439,15 @@ case class TLEdgeParameters(
   // Sanity check the link...
   require (maxTransfer >= slave.beatBytes, s"Link's max transfer (${maxTransfer}) < ${slave.slaves.map(_.name)}'s beatBytes (${slave.beatBytes})")
 
+  def anyDiplomaticClaimsMasterToSlaveAcquireT   = master.anyEmitClaimsAcquireT   || slave.anySupportAcquireT
```

Comment:
I don't think this def is useful.  What would be useful is the && version (so erase the "any" part of this def and use && instead)